### PR TITLE
add testcase for insertAtTheEnd function of LL

### LINKED ISSUE #2: Linked list test case for `insertAtTheEnd`

### PR DESCRIPTION
**Pull request by dishanaik**

### CODE REVIEW

File: test/linked_list/linked_list.test.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 var assert = require('assert')
+const { gunzip } = require('zlib')
```
Comment (sarat1669) [inline test/linked_list/linked_list.test.js]: remove this line

File: test/linked_list/linked_list.test.js [hunk 2]
```diff
@@ -96,6 +97,37 @@ describe('LinkedList', function() {
...
+        assert.strictEqual(ll.tail, ll.head)
+      })
+    })
+    describe('when 1 element is present', function() {
+      beforeEach(function() {
+        ll.insertAtTheBegining(90)
+      })
+      it('should insert after that element', function() {
+        ll.insertAtTheEnd(100)
+        assert.strictEqual(ll.head.next.data, 100)
```
Comment (sarat1669) [inline test/linked_list/linked_list.test.js]:
```suggestion
        assert.strictEqual(ll.getNode(1).data, 100)
```
Might be better

File: test/linked_list/linked_list.test.js [hunk 3]
```diff
@@ -96,6 +97,37 @@ describe('LinkedList', function() {
...
+      })
+    })
+    describe('when multiple elements are present', function() {
+      beforeEach(function() {
+        ll.insertAtTheBegining(8)
+        ll.insertAtTheBegining(88)
+      })
+      it('should insert after that element', function() {
+        ll.insertAtTheEnd(888)
+        assert.strictEqual(ll.head.next.next.data, 888)
```
Comment (sarat1669) [inline test/linked_list/linked_list.test.js]: Same here
```suggestion
        assert.strictEqual(ll.getNode(2), 888)
```

File: test/linked_list/linked_list.test.js (hunk 1 above)
Comment (dishanaik) [reply test/linked_list/linked_list.test.js]: @sarat1669 , this is resolved

File: test/linked_list/linked_list.test.js (hunk 3 above)
Comment (dishanaik) [reply test/linked_list/linked_list.test.js]: @sarat1669 getNode() is not for this purpose. I will change it.

File: test/linked_list/linked_list.test.js (hunk 2 above)
Comment (dishanaik) [reply test/linked_list/linked_list.test.js]: @sarat1669 getNode() is not for this purpose. I will change it.

File: test/linked_list/linked_list.test.js (hunk 3 above)
Comment (dishanaik) [reply test/linked_list/linked_list.test.js]: @sarat1669 , resolved as discussed

**Review by sarat1669 — APPROVED** [review]

**Pull request merged**